### PR TITLE
RHDEVDOCS-4861: Updated GitOps1.7 RN for an inaccuracy

### PR DESCRIPTION
**Pull request by shipsing**
**Aligned team**: Dev Tools

**Purpose**: To resolve the following issue: 
https://issues.redhat.com/browse/RHDEVDOCS-4861

**OCP version this PR applies to**:  enterprise 4.9 and later

**Link to docs preview**:  https://55985--docspreview.netlify.app/openshift-enterprise/latest/cicd/gitops/gitops-release-notes.html#new-features-1-7-0_gitops-release-notes

**Change Management**
Need an ACK from Eng:  @jaideepr97
Need an ACK from PM: @harrietgrace @wtam2018 
Need an ACK from Product experience: @RickJWagner 
Need an ACK from QE: @varshab1210 
Need an ACK from DPM: @pmkovar 
Need an ACK from CS: @Preeticp 

Peer review: @Srivaralakshmi

### CODE REVIEW

**Comment by shipsing** [comment]
@jaideepr97 @harrietgrace @wtam2018 @RickJWagner @varshab1210 @pmkovar @Preeticp, Could you please review this PR and share your feedback? Thanks

**Comment by jaideepr97** [comment]
LGTM thanks @shipsing

**Comment by varshab1210** [comment]
LGTM
Thanks!

**Comment by wtam2018** [comment]
@ishitasequeira Please confirm the correction is correct It changes quite a bit from the original note.

**Comment by ishitasequeira** [comment]
LGTM! Thanks!

**Comment by Preeticp** [comment]
LGTM

File: modules/gitops-release-notes-1-7-0.adoc [hunk 1]
```diff
@@ -40,9 +40,13 @@ In future releases, there is a possibility to deprecate the old method of custom
 
 * With this update, to use the *Environments* feature on the *Developer* tab you must upgrade if you are using a {gitops-title} version prior to 1.7 and {product-title} 4.15 or above. link:https://issues.redhat.com/browse/GITOPS-2415[GITOPS-2415] 
 
-* With this update, applications can be created in any namespace in the same cluster and still managed by the same control-plane’s ArgoCD instance. This is done by adding a new label `argocd.argoproj.io/managed-by-cluster-argocd` to the namespace added in `spec.sourceNamespaces` of the Argo CD custom resource. link:https://issues.redhat.com/browse/GITOPS-2341[GITOPS-2341]
+* With this update, you can create applications, which are managed by the same control plane Argo CD instance, in any namespace in the same cluster. As an administrator, perform the following actions to enable this update: 
+** Add the namespace to the `.spec.sourceNamespaces` attribute for a cluster-scoped Argo CD instance that manages the application
+** Add the namespace to `.spec.sourceNamespaces` in the `AppProject` custom resource that is associated with the application
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-7-0.adoc:45]:
```suggestion
** Add the namespace to the `.spec.sourceNamespaces` attribute in the `AppProject` custom resource that is associated with the application. 
```

File: modules/gitops-release-notes-1-7-0.adoc [hunk 2]
```diff
@@ -40,9 +40,13 @@ In future releases, there is a possibility to deprecate the old method of custom
 
 * With this update, to use the *Environments* feature on the *Developer* tab you must upgrade if you are using a {gitops-title} version prior to 1.7 and {product-title} 4.15 or above. link:https://issues.redhat.com/browse/GITOPS-2415[GITOPS-2415] 
 
-* With this update, applications can be created in any namespace in the same cluster and still managed by the same control-plane’s ArgoCD instance. This is done by adding a new label `argocd.argoproj.io/managed-by-cluster-argocd` to the namespace added in `spec.sourceNamespaces` of the Argo CD custom resource. link:https://issues.redhat.com/browse/GITOPS-2341[GITOPS-2341]
+* With this update, you can create applications, which are managed by the same control plane Argo CD instance, in any namespace in the same cluster. As an administrator, perform the following actions to enable this update: 
+** Add the namespace to the `.spec.sourceNamespaces` attribute for a cluster-scoped Argo CD instance that manages the application
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-7-0.adoc:44]:
```suggestion
** Add the namespace to the `.spec.sourceNamespaces` attribute for a cluster-scoped Argo CD instance that manages the application.
```

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@shipsing PTAL at the suggestions. Thanks!

**Comment by shipsing** [comment]
Thank you @Srivaralakshmi, incorporated your suggestions. Could you please verify and confirm?

**Review by Srivaralakshmi — APPROVED** [review]
@shipsing Approved!

**Pull request merged**